src/openrct2/ParkFile.cpp

```diff
@@ -531,8 +531,8 @@ namespace OpenRCT2
                     cs.ReadWrite(ride.custom_name);
                     cs.ReadWrite(ride.default_name_number);
 
-                    cs.ReadWrite(ride.price);
-                    cs.ReadWrite(ride.price_secondary);
+                    cs.ReadWrite(ride.price[0]);
+                    cs.ReadWrite(ride.price[1]);
```

Comment:
This should write an array now.

Reply:
Like so: `cs.ReadWrite(ride.price);` ?

Also: the rebase meant that using std::begin, end and size on gNewsItems does not work any more, and I'm not sure how to fix it - do you have any idea?

Reply:
Yes.

A shame, you would have to implement a begin and end method with iterators to sort it out. Or maybe you could return just pointers. Otherwise you have to iterate it with an index.